odl/tomo/geometry/parallel.py

```diff
@@ -424,6 +423,12 @@ def __repr__(self):
     # Fix for bug in ABC thinking this is abstract
     rotation_matrix = AxisOrientedGeometry.rotation_matrix
 
+    # TODO: find a more generic place to put this property
```

Comment:
Weird. We rebased on master and didn't get the property. @niinimaki can you check if it's duplicate? If yes, do the interactive rebase and drop the commit.